src/components/AccessibilityBody/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AccessibilityBody = (props) => {
+  const mailto = `mailto:${props.email}`;
+  const emailElement = (<a href={mailto}>{props.email}</a>);
+
+  const communityAccessibilitySite = `${props.communityAccessibilityLink}`;
```

Comment:
I see that `communityAccessibilityLink` is a String in the PropTypes. Can you explain the use of the template string? Does` const communityAccessibilitySite = props.communityAccessibilityLink;` not work?

Reply:
Good catch! I was futzing around with something and can get rid of this. I'll update to:
```
  const communityAccessibilityElement = (
    <a href={props.communityAccessibilityLink}>Website Accessibility Policy</a>
  );
```